### DISCUSSION TITLE
How can Patchman URL be accessed via https:// please?

### DISCUSSION THREAD

**Discussion by OpenThink1** [furlongm/patchman#590] [Q&A] [answered]
Is there where the Patchman web URL be accessed via https:// please?

**Comment by TW1920** [comment]
We're using nginx as reverse proxy for all our services. It's working fine with patchman. You can filter the access with nginx, too.
Other way would be working with Apache, too. Do you know Apache how to work with?

**Reply by OpenThink1** [reply, marked as answer]
Hi, I've managed to access via https://
I used the links for instruction. https://askubuntu.com/questions/73287/how-do-i-install-a-root-certificate.
Many thanks
